06-building-a-program-start-to-finish/writing-tests/output/tests.js

```diff
@@ -0,0 +1,89 @@
+const truebook = require('../example.json');
+const falseBook = require('../package.json');
+
+describe('Tests the invertedIndex.js class: ', () => {
+  const invertedIndexTrue = new InvertedIndex(truebook);
+  const invertedIndexFalse = new InvertedIndex(falseBook);
+
+  describe('Checks if file extension is json', () => {
+    it('should return true for .json file', () => {
+    expect(invertedIndexTrue.checkIfJson('example.json')).toEqual(true);
+    });
+
+    it('should return false for .txt file', () => {
+    expect(invertedIndexTrue.checkIfJson('example.txt')).toEqual(false);
+    });
+  });
+
+  describe('Checks if uploaded file contains title and text', () => {
+    it('should return true for truebook', () => {
+    expect(invertedIndexTrue.containsTitleText()).toEqual(true);
+    });
+
+    it('should return false for falsebook', () => {
+    expect(invertedIndexFalse.containsTitleText()).toEqual(false);
+    });
+  });
+
+  describe('Checks if value returned by getTitlesAndTexts() is an object',
+  () => {
+    const returnedObject = invertedIndexTrue.getTitlesAndTexts();
+
+    it('should return true for instanceof returnedObject', () => {
+    expect(returnedObject instanceof Array).toBeTruthy();
+    });
+  });
+
+  describe('Checks if object returned by getTitlesAndTexts() is of length 2',
```

Comment:
Do you need another 'describe' function block? I believe you can group specifications together in one ‘describe’ function block.